### PR TITLE
#U43 Removed unused configurations and added more meaningful ones

### PR DESCRIPTION
**Pull request by serapieTuyishime**
# PR Description

PR meant to replace the inconsistency in tailwindcss configuration

# Description of tasks that were expected to be completed

- [x] Use one unit of measurement for an object, don't use px and rem together
- [x] Remove unused configurations

# Description of tasks that were expected to be completed and that are not yet completed

- [x] None

# How this maybe tested?

Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

1. run `npm i` to install all required dependences 
2. run `git checkout fix/consistent-tailwind-configuration` to switch branches
3. run `npm run dev`

# Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case its appropriate and applicable

# Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My code generate no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,115 +1,104 @@
...
+                11.1: "3rem",
+                33: "8.25rem",
+            },
+            padding: {
+                2.5: "0.625rem",
+                0.1: "0.1875rem",
+            },
+            boxShadow: {
+                "custom-sm": "1px 2px 5px rgba(0, 0, 0, 0.25)",
+            },
```
Comment (Yvad60) [inline tailwind.config.js:94]: I think these `px` should be removed also

File: components/sections/AboutUs.jsx [hunk 2]
```diff
@@ -1,57 +1,35 @@
...
     const [selectedDiv, setSelectedDiv] = useState(0);
     function changeVisibility(id) {
         if (selectedDiv === id) return setSelectedDiv(null);
         setSelectedDiv(id);
     }
-
     return (
-        <div className="">
+        <section>
             <div className="grid w-full gap-6 mb-6.6 text-xl md:hidden ">
```
Comment (IbrahimBagalwa) [inline components/sections/AboutUs.jsx]: remove the empty space after md:hidden

File: components/sections/AboutUs.jsx [hunk 3]
```diff
@@ -1,57 +1,35 @@
-import Image from "next/image";
 import { useState } from "react";
-import CustomLink from "../ui/CustomLink";
+import { Tab } from "@headlessui/react";
 import { data } from "/utils/aboutUsData";
+import CustomLink from "../ui/CustomLink";
+import UpArrow from "../vectors/UpArrow";
+
+function classNames(...classes) {
```
Comment (chrisostome01) [inline components/sections/AboutUs.jsx]: Is it okay if you add it in the utls folder ?

File: components/sections/AboutUs.jsx [hunk 4]
```diff
@@ -65,55 +43,54 @@ export default function AboutUs() {
...
-                        {data.map((el, index) => {
-                            return (
-                                <div
-                                    className={` relative group overflow-hidden   transition duration-300 pb-1 cursor-pointer ${
-                                        selectedDiv == index
-                                            ? "text-primary-300 "
-                                            : " "
-                                    }`}
+                    <Tab.Group defaultIndex={0}>
+                        <Tab.List className="flex h-8 gap-10.75 text-muted-300 -tracking-lg ">
```
Comment (IbrahimBagalwa) [inline components/sections/AboutUs.jsx]: even here remove the empty space after `-tracking-lg`

**Review by chrisostome01 — CHANGES_REQUESTED** [review]
Well done @serapieTuyishim, here is comment you can work on

File: components/sections/AboutUs.jsx [hunk 5]
```diff
@@ -65,55 +43,54 @@ export default function AboutUs() {
...
+                            {data.map((el, index) => (
+                                <Tab
                                     key={index}
-                                    onClick={() => setSelectedDiv(index)}
+                                    className={({ selected }) =>
+                                        classNames(
+                                            "outline-none after:opacity-0 transition-all duration-500 after:bg-primary-300 after:w-full after:h-0.1 after:transition-all after:duration-500 after:ease-in after:absolute after:bottom-0 ",
+                                            "relative group overflow-hidden   transition duration-300 pb-1 cursor-pointer",
+                                            selected
+                                                ? "after:!left-0 after:opacity-100 text-primary-300 "
```
Comment (IbrahimBagalwa) [inline components/sections/AboutUs.jsx]: `after:bottom-0` and `text-primary-300` here as well

File: components/sections/AboutUs.jsx [hunk 6]
```diff
@@ -65,55 +43,54 @@ export default function AboutUs() {
...
+                    <Tab.Group defaultIndex={0}>
+                        <Tab.List className="flex h-8 gap-10.75 text-muted-300 -tracking-lg ">
+                            {data.map((el, index) => (
+                                <Tab
                                     key={index}
-                                    onClick={() => setSelectedDiv(index)}
+                                    className={({ selected }) =>
+                                        classNames(
+                                            "outline-none after:opacity-0 transition-all duration-500 after:bg-primary-300 after:w-full after:h-0.1 after:transition-all after:duration-500 after:ease-in after:absolute after:bottom-0 ",
+                                            "relative group overflow-hidden   transition duration-300 pb-1 cursor-pointer",
```
Comment (IbrahimBagalwa) [inline components/sections/AboutUs.jsx]: much space between `overflow-hidden` and `transition`

File: components/sections/AboutUs.jsx [hunk 7]
```diff
@@ -65,55 +43,54 @@ export default function AboutUs() {
...
                                 >
-                                    <div>
-                                        {el.paragraphs[0]}{" "}
-                                        <CustomLink
-                                            link={el.link}
-                                            text="Learn more"
-                                            style="text-2xl"
-                                        />
+                                    <div
+                                        className={`gap-6 grid transition-all  duration-700`}
```
Comment (IbrahimBagalwa) [inline components/sections/AboutUs.jsx]: `transition-all` and `duration-700` much spaces between both classes

**Review by IbrahimBagalwa — CHANGES_REQUESTED** [review]
### Great Job @serapieTuyishime we request few changes.

**Review by Yvad60 — CHANGES_REQUESTED** [review]
Just one thing to change otherwise you did a great job

File: components/sections/AboutUs.jsx (hunk 3 above)
Comment (serapieTuyishime) [reply components/sections/AboutUs.jsx]: Yeah good idea

**Review by Yvad60 — APPROVED** [review]
Approved now!

File: components/sections/AboutUs.jsx (hunk 2 above)
Comment (serapieTuyishime) [reply components/sections/AboutUs.jsx]: thanks

**Review by chrisostome01 — APPROVED** [review]
LGTM 👍

**Review by IbrahimBagalwa — APPROVED** [review]
## Great Job 👍 

LGTM 💯

**Pull request merged**